p9-presentation/sections/_methodology.qmd

```diff
@@ -1,7 +1,494 @@
 Presenter: Christian Bager Bach Houmann.
 
-## PLS1-SM
+::: {.notes}
+My name is Christian, and I'll be presenting our replica of the MOC pipeline.<br>
+This is what we used to establish baselines for our experiments.
+:::
 
-## ICA
+## MOC Pipeline {auto-animate="true"}
+::: {style="width=100%;"}
+![](/static/methodology/pipeline.png){fig-align="center" width="35%"}
+:::
+::: {.fragment data-id="moc-box"}
+<div style="position: absolute; top: 34.45%; left: 30.5%; width: 18%; height: 35.25%; border: 2px solid red; box-sizing: border-box; pointer-events: none; border-radius: 10px;"></div>
+:::
+
+::: {.fragment data-id="moc-box"}
+<div style="position: absolute; top: 34.45%; left: 51.25%; width: 18%; height: 35.25%; border: 2px solid red; box-sizing: border-box; pointer-events: none; border-radius: 10px;"></div>
+:::
+
+::: {.notes}
+This is the entire MOC pipeline at inference-time.<br>
+<br>
+We have attempted to replicate this pipeline as faithfully as we could, but have had to make some design choices that differ from the original --- mostly due to a lack of information regarding some aspects of the original design.<br>
+<br>
+It's important to keep in mind that this pipeline is for multivariate regression.<br>
+That means we create predictions for each of 8 major oxides.<br>
+However, most of the steps actually involve data being processed for each oxide individually.<br>
+For both ICA and PLS, you can imagine that after some preprocessing steps, we branch out into 8 branches, each of which predict its own values for its associated oxide.<br>
+And in the end, we put the outputs together to produce the MOC prediction.<br>
+<br>
+I start by walking through the PLS-SM side, and then the ICA side.
+:::
+
+<!-- ## PLS-SM 
+- Recreating the PLS-SM method presented by @cleggRecalibrationMarsScience2017 and @andersonImprovedAccuracyQuantitative2017
+    - Preprocessing
+    - Outlier removal
+    - Training
+    - Inference via submodels
+
+::: {.notes}
+Starting with the PLS-SM method, we have attempted to recreate the system presented by both Clegg and Anderson et al.
+::: -->
+
+## PLS-SM Motivation {.unlisted auto-animate="true"}
+- **PLS**
+    - Good for chemometrics
+    - Readily intrepretable
+
+::: {.notes}
+Let me start by explaining what motivated PLS-SM, because that will help us understand why it does what it does.<br>
+<br>
+First: they chose PLS because it can handle noisy data with many more variables than observations, and with significant correlation between variables.<br>
```

Comment:
```suggestion
First: they chose PLS because it can handle noisy data with far more variables than observations, and with significant correlation between variables.<br>
```